src/app/locale/translations/en.json

```diff
@@ -236,7 +236,7 @@
             "instrument_manage_modal_confirmation_label": "Are you sure you want to delete this stored payment method?",
             "instrument_manage_modal_title_text": "Manage stored payment methods",
             "instrument_manage_modal_empty_text": "You do not have any stored payment methods.",
-            "instrument_manage_table_header_ending_in_text": "Ending in",
+            "instrument_manage_table_header_ending_in_text": "Ending in ",
```

Comment:
This is not what I meant, I would rather take a look at `ManageAccountInstrumentsTable.tsx`